pandas/core/categorical.py

```diff
@@ -548,10 +548,6 @@ def _validate_categories(cls, categories, fastpath=False):
 
         if not fastpath:
 
-            # Categories cannot contain NaN.
```

Comment:
I feel like the Index, `Index([None, u'A', u'B'], dtype='object')` needs to be passed to `Categorical` when doing `MultiIndex`, as when `dropna=False`, `None` could also be the index/column name. Or I didn't get this correctly?